<launch>
  
  <arg name="bag_manager_param"  default="$(find-pkg-share bag_manager_py)/config/bag_manager.param.yaml"/>
  <!-- bag_manager node -->
  <node pkg="bag_manager_py" 
        exec="bag_manager_node"
        name="bag_manager_node"
        output="screen">

    <!-- パラメータ -->
    <param from="$(var bag_manager_param)"/>
    <!-- トピックのremap -->
    <remap from="/rosbag2_recorder/trigger" to="/rosbag2_recorder/trigger"/>
  </node>


</launch>